src/soulmates/index.scss

```diff
@@ -0,0 +1,4 @@
+@import '_core';
+@import '_adverts';
+@import '_adverts-soulmates';
+@import '_advert';
```

Comment:
I can't see any changes from when it was the penultimate stylesheet getting imported, but I've moved it to the bottom.